db/migrate/20210415163231_add_notify_fields_to_nomination_email.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class AddNotifyFieldsToNominationEmail < ActiveRecord::Migration[6.1]
+  def change
+    change_table :nomination_emails, bulk: true do
+      add_column :nomination_emails, :notify_id, :string
+      add_column :nomination_emails, :delivered_at, :datetime
+    end
+
+    add_index :nomination_emails, :notify_id
+  end
+end
```

Comment:
In a dream world we would do single table inheritance on them, buuut I think that's probably alright.

Reply:
Just what I was thinking too but we can do that in future when things have calmed down